spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done do
+      status { 2 }
```

Comment:
2だと何を示しているのか分からないのでSymbolで記載してください
```suggestion
      status { :done }
```

Reply:
Symbolで記載しました。
d17a4ee